spec/forms/update_user_password_form_spec.rb

```diff
@@ -80,14 +82,52 @@
 
         expect(encryptor).to have_received(:call)
       end
+
+      it 'logs that the user has an active profile' do
+        result = subject.submit(params)
+
+        expect(result.extra).to include(
+          active_profile_present: true,
+          pending_profile_present: false,
+        )
+      end
+    end
+
+    context 'the user has a pending profile' do
+      let(:profile) { create(:profile, :verify_by_mail_pending, :verified, pii: { ssn: '1234' }) }
+      let(:user) { profile.user }
+
+      it 'does not call ActiveProfileEncryptor' do
+        expect(ActiveProfileEncryptor).to_not receive(:new)
+
+        subject.submit(params)
+      end
+
+      it 'logs that the user does not have an active or pending profile' do
```

Comment:
Nit: this description is wrong

```suggestion
      it 'logs that the user has a pending profile but not an active one' do
```

Reply:
whoops, copy/paste issue. willfix